src/test/java/com/synopsys/integration/stepworkflow/StepWorkflowTest.java

```diff
@@ -17,26 +22,62 @@
     public static final Integer TEST_DATA_FOUR = 4;
     public static final Integer TEST_DATA_FIVE = 5;
 
-    private static final String exceptionMessage1 = "Exception #1";
-    private static final IntegrationException exception = new IntegrationException(exceptionMessage1);
+    private static final IntegrationException integrationException = new IntegrationException("Exception #1");
 
-    private final SubStep<Object, Integer> successfulSupplierOne = SubStep.ofSupplier(this::successfulDataSupplierOne);
-    private final SubStep<Object, Integer> successfulSupplierTwo = SubStep.ofSupplier(this::successfulDataSupplierTwo);
-    private final SubStep<Object, Integer> successfulSupplierThree = SubStep.ofSupplier(this::successfulDataSupplierThree);
-    private final SubStep<Object, Integer> unsuccessfulSupplierOne = SubStep.ofSupplier(this::unsuccessfulDataSupplier);
+    private SubStep<Object, Integer> subStepSupplierOne;
+    private SubStep<Object, Integer> subStepSupplierTwo;
+    private SubStep<Object, Integer> subStepSupplierThree;
+    private SubStep<Object, Object> subStepExecutorOne;
+    private SubStep<Integer, Object> subStepFunctionOne;
+    private SubStep<Integer, Object> subStepFunctionTwo;
+    private StepWorkflow.FlowController<Object, Integer> flowControllerOne;
+    private StepWorkflow.Builder<Integer> successfulBuilderOfSupplierOne;
+    private StepWorkflow.Builder<Integer> builderThenSubStepSupplierTwo;
 
-    private final SubStep<Object, Object> successfulExecutorOne = SubStep.ofExecutor(this::successfulExecutor);
-    private final SubStep<Integer, Object> successfulFunctionOne = SubStep.ofFunction(this::successfulFunctionOne);
-    private final SubStep<Integer, Object> successfulFunctionTwo = SubStep.ofFunction(this::successfulFunctionTwo);
+    private StepWorkflow.Builder<Integer> unsuccessfulBuilderOfSupplierOne;
+    private SubStep<Object, Integer> unsuccessfulSubStepSupplierOne;
```

Comment:
Outside of subStepSupplierOne and subStepSupplierTwo, the serial naming convention for these variables doesn't really seem to be doing us any favors-- they are loosely order-dependent, but we're never creating a StepWorkflow that's just Step One, Step Two, Step Three.

I'd consider renaming these variables to imply their implementation.